src/tests.rs

```diff
@@ -455,6 +484,39 @@ fn test_interests() {
     );
 }
 
+fn test_user_span() {
+    let p = g::Pipeline::new();
+
+    let span = tracing::error_span!("pipeline span", pipeline = true);
+    unsafe {
+        p.set_data("tracing-gstreamer:span", span);
+    }
+
+    let p_addr = p.as_object_ref().to_glib_none().0 as usize;
+    MockSubscriber::with_expected(
+        |m| m.target() == "gstreamer::test_object_cat",
+        "test_with_object",
+        move || {
+            let cat = g::DebugCategory::new("test_object_cat", g::DebugColorFlags::empty(), None);
+            g::error!(cat, obj: &p, "with object");
+        },
+        vec![Expect::GstEvent(GstEvent {
+            message: "with object",
+            kvs: KV {
+                gobject_address: Some(p_addr),
+                gobject_type: Some("GstPipeline"),
+                gstobject_name: Some("pipeline1"),
+                gstelement_state: Some("null"),
+                gstelement_pending_state: Some("void-pending"),
+                ..Default::default()
+            },
+            level: Level::ERROR,
+            target: "test_object_cat",
+            has_parent: true, // FIXME
```

Comment:
Describe what this FIXME is about :)

Reply:
Got it, I’m interested why this might be happening too, so I will play with this too…

Reply:
Cool, thanks.

I rebased the branch and it no longer depends on the `gstreamer-rs` main so you can easily give it shot.